### PR TITLE
Add Intercom chat

### PR DESCRIPTION
**Pull request by vsaarinen**
Chat was actually enabled in Intercom's side – our Intercom setup already supported it. This PR cleans up the code, enables Intercom on staging, and moves the Intercom IDs to environment variables.

### CODE REVIEW

File: src/js/intercom.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
+let loggedInUserEmail: string| undefined;
```
Comment (dennari) [inline src/js/intercom.ts]: What's going on here?. Should'nt this be stored in the Redux state like everything else? Also missing space.

File: src/js/intercom.ts [hunk 2]
```diff
@@ -30,6 +33,6 @@ export function trackEvent(event: string) {
 export function update(options?: any) {
   const intercom = (window as any).Intercom;
   if (intercom) {
-    intercom('update', options);
+    intercom('update', { email: loggedInUserEmail, ...options });
```
Comment (dennari) [inline src/js/intercom.ts]: Should check for `undefined`

Comment (vsaarinen) [inline src/js/intercom.ts]: Some documentation says that it's fine to call 'update' without any parameters, so technically it shouldn't matter if it's called with an undefined email address. But I can add the check.

**Comment by dennari** [comment]
:+1:

**Pull request merged**